IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CommunicationsApplianceTypeOpticalLineTerminal/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties in this property set are applicable to the optical line terminal type of communications appliance.
```

Comment:
Case for usage.

Reply:
If we have an applicability to PDT .USERDEFINED. we need to further specify it. Meaning we need to set the proper usage for this particular pset with `Object Predefined Type` and `Property Sets for Objects`. Applicability on .USERDEFINED. is too vague because it could be anything.